source/platforms/bitbucket_cloud/BitBucketCloudAPI.ts

```diff
@@ -214,11 +214,12 @@ export class BitBucketCloudAPI {
     return await res.text()
   }
 
-  getDangerComments = async (dangerID: string): Promise<BitBucketCloudPRComment[]> => {
+  getDangerMainComments = async (dangerID: string): Promise<BitBucketCloudPRComment[]> => {
```

Comment:
`getDangerComments` is a part of the interface for all API providers, so you probably shouldn't modify this implementation & rename it.

Would you mind either putting a helper in the BitBucketCloud.ts file (since that's the only place that is calling this `getDangerMainComments` method and/or duplicating the implementation (with your changes) into a new method?

Reply:
I don't think `getDangerComments` is a part of the interface because we only have this function in `BitBucketServerAPI` and `BitBucketCloudAPI`. 

I believe the name `getDangerComments` would confused us, since we don't know if the result would be main comment or inline comment.

Moreover, we don't really have the usecase that we need both main comments, and inline comments at the same time, so I renamed it to `getDangerMainComments`

Reply:
This felt off to me also, but I think I see your reasoning 👍🏻

Reply:
Sounds good! Sorry that i didn't dive deeper to verify my comments!